impl/src/planning/subplanners.rs

```diff
@@ -366,13 +314,40 @@ impl<'planner> CrateSubplanner<'planner> {
 
     filtered_deps.sort();
 
+    let workspace_member_dependents: Vec<PathBuf> = self
+      .crate_catalog_entry
+      .workspace_member_dependents
+      .iter()
+      .filter_map(|pkg_id| {
+        if let Some(workspace_member) = self
+          .crate_catalog
+          .metadata
+          .packages
+          .iter()
+          .find(|pkg| pkg.id.eq(pkg_id))
```

Comment:
Same as elsewhere is there a reason this is not `==` am I missing something or being dumb?

Reply:
I found this to be clearer than de-referencing references to allow for `==` but that may be personal experience. I've since updated these.